lib/statsd/instrument/metric.rb

```diff
@@ -93,6 +94,14 @@ def inspect
     s:  'set',
   }
 
+  # Strip metric names of special characters used by StatsD line protocol, replace with underscore
+  #
+  # @param name [String]
+  # @return [String]
+  def self.normalize_name(name)
+    name.tr(':|', '_')
+  end
```

Comment:
Any thoughts on making this method a private instance method? I don't see how it could be useful to anyone to use it outside of the Metric class.

Maybe normalize_tags could be made a private instance method at some point, but that would break compatibility with previous versions.